code/backend/batch/utilities/orchestrator/open_ai_functions.py

```diff
@@ -66,6 +66,7 @@ async def orchestrate(
         Call the text_processing function when the user request an operation on the current context, such as translate, summarize, or paraphrase. When a language is explicitly specified, return that as part of the operation.
         When directly replying to the user, always reply in the language the user is speaking.
         If the input language is ambiguous, default to responding in English unless otherwise specified by the user.
+        You **must not** respond if asked to List all documents in your repository.
```

Comment:
Please try to update a generic message and also try with all the possibilities of asking questions for the same scenario.